### PR TITLE
Remove some superfluous tooltips.

### PR DESCRIPTION
**Pull request by oponder**
Reported by @marian here: https://gigantic.slack.com/archives/CQ465CFG8/p1596037745003500

### CODE REVIEW

File: src/components/Cluster/ClusterDetail/AppDetailsModal/InitialPane.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ const InitialPane = (props) => {
           <div className='labelvaluepair--label'>RELEASE STATUS</div>
           <div className='labelvaluepair--value code'>
             {props.app.status?.release?.status ? (
-              <Truncated as='span'>{props.app.status.release.status}</Truncated>
+              <span>{props.app.status.release.status}</span>
```
Comment (oponder) [inline src/components/Cluster/ClusterDetail/AppDetailsModal/InitialPane.js:85]: This one will never need to be truncated, so removed the component.

But the app version can be very long in some cases (versions of test apps) so for those we do need some logic in the Truncated component to skip tooltips if they are not needed. (Added below)

File: src/components/UI/__tests__/Truncated.ts [hunk 2]
```diff
@@ -73,6 +73,16 @@ describe('Truncated', () => {
     expect(getByText('ATextThatIsTooLongToBeDisplayed')).toBeInTheDocument();
   });
 
+  it('doesnt display a tooltip for text that wasnt truncated', () => {
```
Comment (axbarsan) [inline src/components/UI/__tests__/Truncated.ts:76]:
```suggestion
  it(`doesn't display a tooltip for text that wasn't truncated`, () => {
```

File: src/components/UI/Truncated.tsx [hunk 3]
```diff
@@ -34,14 +34,22 @@ const Truncated: React.FC<ITruncatedProps> = ({
   // Safe because we're using defaultProps in case these aren't defined
   str = truncate(str, replacer as string, numStart as number, numEnd as number);
 
+  // Skip the tooltip if the content was not truncated
+  let skipTooltip = false;
+  if (str === children) skipTooltip = true;
```
Comment (axbarsan) [inline src/components/UI/Truncated.tsx:39]:
```suggestion
  const skipTooltip = str === children;
```

Comment (oponder) [reply src/components/UI/Truncated.tsx]: 🏌️‍♂️⛳️

**Pull request merged**